enable flake8 configuration from pyproject.toml (#134)
microsoft/vscode-flake8: README.md

If you want to disable flake8, you can [disable this extension](https://code.visualstudio.com/docs/editor/extension-marketplace#_disable-an-extension) per workspace in Visual Studio Code.

## Configuration

> `flake8` can be configured with either a `pyproject.toml` or [`.flake8`](https://flake8.pycqa.org/en/latest/user/configuration.html) file. The extension will first check for the configuration file in the root directory, either the `pyproject.toml`, before checking the `.flake8` file. You can specific a custom location using the flake8.args as either `--toml-config=/path/to/pyproject.toml` or `--config=/path/to/.flake8`. While flake8 does not natively support the pyproject.toml, support is provided by the [flake8-pyproject](https://pypi.org/project/Flake8-pyproject/)) extension.

Reviewer: ```suggestion
`flake8` can be configured with either a `pyproject.toml` or [`.flake8`](https://flake8.pycqa.org/en/latest/user/configuration.html) file. `flake8` will first check for the configuration file in the root directory, either the `pyproject.toml`, before checking the `.flake8` file. You can specifiy a custom location using the flake8.args as either `--toml-config=/path/to/pyproject.toml` or `--config=/path/to/.flake8`. While flake8 does not natively support the pyproject.toml, support is provided by the [flake8-pyproject](https://pypi.org/project/Flake8-pyproject/) plugin included in the extension.
```

> `flake8` can be configured with either a `pyproject.toml` or [`.flake8`](https://flake8.pycqa.org/en/latest/user/configuration.html) file. `flake8` will first check for the configuration file in the root directory, either the `pyproject.toml`, before checking the `.flake8` file. You can specifiy a custom location using the flake8.args as either `--toml-config=/path/to/pyproject.toml` or `--config=/path/to/.flake8`. While flake8 does not natively support the pyproject.toml, support is provided by the [flake8-pyproject](https://pypi.org/project/Flake8-pyproject/) plugin included in the extension.

Reviewer 2: just to make sure I understand it right, does flake8 check for pyproject.toml before checking for the .flake8 file? If so, I'd only suggest changing the second sentence to something like "`flake8` will check for a configuration file in the root directory. The `pyproject.toml` file takes precedence over `.flake8`."  (or something like that).

Reviewer 2: Sounds good. Then maybe something like:
> "`flake8` can be configured with either a `pyproject.toml` or [`.flake8`](https://flake8.pycqa.org/en/latest/user/configuration.html) file located in your project's root directory"

Thoughts?

Author: ```suggestion
`flake8` can be configured with either a `pyproject.toml` or [`.flake8`](https://flake8.pycqa.org/en/latest/user/configuration.html) file. `flake8` will first check for the configuration file in the root directory, first checking the `pyproject.toml`, before checking the `.flake8` file. You can specify a custom location using the flake8.args as either `--toml-config=/path/to/pyproject.toml` or `--config=/path/to/.flake8`. While flake8 does not natively support the pyproject.toml, support is provided by the [flake8-pyproject](https://pypi.org/project/Flake8-pyproject/) plugin included in the extension.
```

> `flake8` can be configured with either a `pyproject.toml` or [`.flake8`](https://flake8.pycqa.org/en/latest/user/configuration.html) file. `flake8` will first check for the configuration file in the root directory, first checking the `pyproject.toml`, before checking the `.flake8` file. You can specify a custom location using the flake8.args as either `--toml-config=/path/to/pyproject.toml` or `--config=/path/to/.flake8`. While flake8 does not natively support the pyproject.toml, support is provided by the [flake8-pyproject](https://pypi.org/project/Flake8-pyproject/) plugin included in the extension.

Reviewer 2: ```suggestion
`flake8` can be configured with either a `pyproject.toml` or [`.flake8`](https://flake8.pycqa.org/en/latest/user/configuration.html) file. `flake8` will check for the configuration file in the root directory, first checking the `pyproject.toml`, before checking the `.flake8` file. You can specify a custom location using the flake8.args as either `--toml-config=/path/to/pyproject.toml` or `--config=/path/to/.flake8`. While flake8 does not natively support the pyproject.toml, support is provided by the [flake8-pyproject](https://pypi.org/project/Flake8-pyproject/) plugin included in the extension.
```

> The following is an example configuration for `flake8` in the `pyproject.toml`.

Reviewer 2: ```suggestion
The following is an example configuration for `flake8` in a `pyproject.toml` file.
```
